TwilioVerifySDKTests/TwilioVerify/Sources/Domain/Factor/FactorMapperTests.swift

```diff
@@ -121,7 +121,7 @@ class FactorMapperTests: XCTestCase {
     let expectedFactorResponse = [Constants.friendlyNameKey: Constants.expectedFriendlyName,
                                   Constants.accountSidKey: Constants.expectedAccountSid]
     let data = try! JSONSerialization.data(withJSONObject: expectedFactorResponse, options: .prettyPrinted)
-    let factorPayload = CreateFactorPayload(friendlyName: Constants.friendlyNameValue, type: Constants.pushType, serviceSid: Constants.serviceSidValue,
+    let factorPayload = CreateFactorPayload(friendlyName: Constants.friendlyNameValue, type: Constants.pushType, allowIphoneMigration: false, serviceSid: Constants.serviceSidValue,
                                             identity: Constants.identityValue, config: [:], binding: [:], accessToken: Constants.accessToken, metadata: nil)
     XCTAssertThrowsError(try mapper.fromAPI(withData: data, factorPayload: factorPayload)) { error in
       XCTAssertEqual((error as! TwilioVerifyError).errorDescription, TwilioVerifyError.mapperError(error: NSError()).errorDescription)
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/1230259fcee173b2bad2b801de0a522beecabdd9/TwilioVerifySDKTests/TwilioVerify/Sources/Domain/Factor/FactorMapperTests.swift#L127"></span>Discouraged direct initialization of types that can be harmful<br />
<code>discouraged_direct_init</code> <code>FactorMapperTests.swift:127</code></td>
    </tr>
  </tbody>
</table>